src/mopidy_mpd/protocol/stored_playlists.py

```diff
@@ -24,7 +53,7 @@ def _get_playlist(context, name, *, must_exist=True):
 
 
 @protocol.commands.add("listplaylist")
-def listplaylist(context, name):
+def listplaylist(context: MpdContext, name: str) -> protocol.Result:
```

Comment:
```suggestion
def listplaylist(context: MpdContext, name: str) -> protocol.ResultList:
```

Reply:
`ResultList` is part of `Result`. The only reason `listplaylists()` is typed explicitly to return `ResultList` is because it is called from `lsinfo()` which needs that specific type.